File: pset1/mario.c
#include<stdio.h>
#include<cs50.h>

int main()
{
    int n, i, j, k;
    do
    {
        printf("height:");
        n = get_int();
    }
    while (!(n > -1 && n < 24));


    for (i = 0; i < n; i++)
    {
        for (k = n - i - 1; k > 0; k--)
        {
            printf(" ");
        }
        for (j = i + 1; j > 0; j--)
        {
            printf("#");
        }

        printf("  ");

        for (j = i + 1; j > 0; j--)
        {
            printf("#");
        }

        printf("\n");
    }

    return 0;
}

File: pset6/mario.py
from cs50 import get_int

while True:

    height=get_int("Height:")
    if height > -1 and height < 24:
        break
scd ~/workspace/pset6/hello/ && submit50 cs50/2018/x/sentimental/hello

"""for i in range(height):
    #for k in range((height-i-1),0):
    #    print(" ")
    for j in range((i+1),0):
        print("#")
    print()"""

for i in range(height):
     print(" "*(height-i-1),end="")
     print("#"*(i+1))